model_api/python/openvino/model_api/models/utils.py

```diff
@@ -14,6 +14,8 @@
  limitations under the License.
 """
 
+from __future__ import annotations
```

Comment:
Why do we need this?

Reply:
For python versions below `3.10` we can't use ` np.ndarray | None` notation without `__future__`. Otherwise we will have to use `Union[np.ndarray, None]`. This just looks cleaner 🙂